code/Player/Cameras/FollowEntityCamera.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Sandbox;
+
+namespace TTT;
+
+public partial class FollowEntityCamera : CameraMode, ISpectateCamera
+{
+	[Net, Local] public Entity FollowedEntity { get; set; }
```

Comment:
`[Net, Local]` should be on their own line, add a newline between public and private vars. This is just our coding style.